cmd/orphan-finder/main.go

```diff
@@ -91,9 +91,19 @@ func (t orphanType) String() string {
 	}
 }
 
+// The official supported format of orphaned cert log lines is:
+// `[AUDIT] Failed RPC to store at SA, orphaning [pre]certificate: serial=[\w+], cert=[\w+], err=[.+], issuerID=[\d+], regID=[\d+], orderID=[\d+]`
```

Comment:
This comment suggests that the ordering of fields is important, but it is not (per the refactored code below). Also the order given in the example is different than the updated log line in ca.go, which moves `err` to the end of the line (a nice change, IMO).